portality/lib/dates.py

```diff
@@ -58,3 +58,11 @@ def random_date(fro=None, to=None):
     span = int((to - fro).total_seconds())
     s = randint(0, span)
     return format(to - timedelta(seconds=s))
+
+
+def before(timestamp, seconds):
```

Comment:
Do you really mean timestamp here? I think of timestamps as strings, which you can't do arithmetic on. You'd need a ```datetime``` object.

Reply:
It is a datetime object, yeah.  I think of a timestamp as simply a representation of the time, in whatever form.  I poked around the internet for alternative ideas, but in some languages Timestamp is an actual object, so I'm convinced that it doesn't /have/ to be a string :)